app_inference_engine/src/tflm_conf.h

```diff
@@ -0,0 +1,14 @@
+#ifndef _tflm_conf_h_
+#define _tflm_conf_h_
+
+#define TFLM_OPERATORS 25
```

Comment:
if this allocates memory for the operators then you might want a note pointing them to this as they are likely to exceed it